overrides/scripts/RFTools.zs

```diff
@@ -233,6 +233,7 @@ recipes.addShaped(<rftools:orphaning_card>, [[null,<enderio:item_alloy_ingot>,nu
 # Powercell Card
 recipes.remove(<rftools:powercell_card>);
 recipes.addShaped(<rftools:powercell_card>, [[<enderio:item_alloy_ingot:3>,<enderio:item_alloy_ingot:1>,<enderio:item_alloy_ingot:3>],[<enderio:item_alloy_ingot:1>,<enderio:item_alloy_ingot:3>,<enderio:item_alloy_ingot:1>],[<enderio:item_alloy_ingot:3>,<enderio:item_alloy_ingot:1>,<enderio:item_alloy_ingot:3>]]);
+recipes.addShapeless(<rftools:powercell_card>, [<rftools:powercell_card>]);
```

Comment:
`addHiddenShapeless` & add a name

Reply:
You want a tooltip added?

Reply:
naw. any missing clearing recipe should be considered an issue, and adding a tooltip for something that anything that has data like this can do is redundant.

Reply:
sounds good